recipes/linkstats/meta.yaml

```diff
@@ -18,7 +18,7 @@ build:
 requirements:
   build:
     - {{ compiler('cxx') }}
-    - clang >=12.0
+    - clang >=11.0
```

Comment:
On OSX clang is used anyway, so maybe put `# [linux]` on this and add ` - compiler('c')  # [osx]`